### PR TITLE
Fix new feature slide

### PR DESCRIPTION
**Pull request by cmgustavo**

### CODE REVIEW

File: src/providers/app/app.ts [hunk 1]
```diff
@@ -146,10 +146,14 @@ export class AppProvider {
...
-    );
+    const previousVersion =
+      parseInt(version.major, 10) +
+      parseInt(version.minor, 10) +
+      parseInt(version.patch, 10);
+    const currentVersion =
+      parseInt(targetVersion.major, 10) +
+      parseInt(targetVersion.minor, 10) +
+      parseInt(targetVersion.patch, 10);
+    return previousVersion < currentVersion;
```
Comment (JohnathanWhite) [inline src/providers/app/app.ts:157]: should be <=

File: src/providers/app/app.ts [hunk 2]
```diff
@@ -146,10 +146,14 @@ export class AppProvider {
       return false;
     }
 
-    return (
-      version.major === targetVersion.major &&
-      version.minor === targetVersion.minor &&
-      version.patch === targetVersion.patch
-    );
+    const previousVersion =
```
Comment (JohnathanWhite) [inline src/providers/app/app.ts:149]: would rename this targetVersion

File: src/pages/home/home.ts [hunk 3]
```diff
@@ -215,7 +215,7 @@ export class HomePage {
               }
             });
           } else {
-            this.persistenceProvider.setNewFeatureSlidesFlag(currentVs);
+            this.persistenceProvider.setNewFeatureSlidesFlag(dismissFlag);
```
Comment (JohnathanWhite) [inline src/pages/home/home.ts:218]: good catch

File: src/providers/app/app.ts (hunk 1 above)
Comment (cmgustavo) [reply src/providers/app/app.ts]: Fixed. Thanks!

**Review by JohnathanWhite — APPROVED** [review]

**Pull request merged**